examples/landing-zone-v2/tier3/dns/dns.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+# DNS A Record Example
+#########
+# A Record example for Client DNS subzone
+# SC-22
+apiVersion: dns.cnrm.cloud.google.com/v1beta1
+kind: DNSRecordSet
+metadata:
+  name: application1-client-name-standard-core-public-dns-a-rset # kpt-set: application1-${client-name}-standard-core-public-dns-a-rset
```

Comment:
these examples are still a work in progress on how we'll organize them, but for what it's worth, PR #400 has a common Kptfile/setters.yaml to try to avoid duplication
it also has a `workload-name` which could be used instead of `application1`

we could cleanup/reorganize in a later PR

Reply:
I updated to `workload-name`, but for the setters, if we merge your PR first then we can address the setters.